<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>AES &amp; SHA-256</string>
    </property>
    <widget class="QTextEdit" name="lineEditInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>621</width>
       <height>66</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEncrypt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>621</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="lineEditOutput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>621</width>
       <height>66</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDecrypt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>621</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="lineEditHashInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>621</width>
       <height>66</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonHash">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>430</y>
       <width>621</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Hash </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="lineEditHashOutput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>480</y>
       <width>621</width>
       <height>66</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="Line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>305</y>
       <width>751</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
